lightning/src/ln/functional_test_utils.rs

```diff
@@ -2225,12 +2230,14 @@ pub fn expect_payment_forwarded<CM: AChannelManager, H: NodeHolder<CM=CM>>(
 }
 
 macro_rules! expect_payment_forwarded {
-	($node: expr, $prev_node: expr, $next_node: expr, $expected_fee: expr, $upstream_force_closed: expr, $downstream_force_closed: expr) => {
+	($node: expr, $prev_node: expr, $next_node: expr, $expected_fee: expr,
+	 $expected_extra_fee_limit_msat: expr, $upstream_force_closed: expr,
+	 $downstream_force_closed: expr) => {
 		let mut events = $node.node.get_and_clear_pending_events();
 		assert_eq!(events.len(), 1);
 		$crate::ln::functional_test_utils::expect_payment_forwarded(
 			events.pop().unwrap(), &$node, &$prev_node, &$next_node, $expected_fee,
-			$upstream_force_closed, $downstream_force_closed);
+			$expected_extra_fee_limit_msat, $upstream_force_closed, $downstream_force_closed);
```

Comment:
IMO, we should only add this new parameter to the function `expect_payment_forwarded` and not the macro because most tests won't use it.